src/main/scala/nl.knaw.dans.easy.stage/dataset/EMD.scala

```diff
@@ -17,27 +17,20 @@ package nl.knaw.dans.easy.stage.dataset
 
 import java.io.File
 import java.net.URI
-import java.nio.charset.StandardCharsets
-import java.nio.file.{ Files, Paths }
 
 import nl.knaw.dans.easy.stage.Settings
 import nl.knaw.dans.easy.stage.lib.Util._
 import nl.knaw.dans.lib.logging.DebugEnhancedLogging
-import nl.knaw.dans.lib.string._
 import nl.knaw.dans.pf.language.ddm.api.Ddm2EmdCrosswalk
 import nl.knaw.dans.pf.language.emd.EasyMetadata
 import nl.knaw.dans.pf.language.emd.binding.EmdMarshaller
-import nl.knaw.dans.pf.language.emd.types.{ BasicIdentifier, BasicRemark, EmdArchive, EmdConstants }
-import org.apache.commons.lang.BooleanUtils
+import nl.knaw.dans.pf.language.emd.types.{ BasicIdentifier, EmdArchive, EmdConstants }
 
 import scala.util.{ Failure, Success, Try }
-import scala.xml.{ Elem, XML }
 
 object EMD extends DebugEnhancedLogging {
 
-  private val depositorInfoDir = Paths.get("metadata/depositor-info")
-
-  def create(sdoDir: File)(implicit s: Settings): Try[EasyMetadata] = {
+  def create(sdoDir: File, acceptedLicense: Boolean)(implicit s: Settings): Try[EasyMetadata] = {
```

Comment:
Maybe rename to

```suggestion
  def create(sdoDir: File, licenseAccepted: Boolean)(implicit s: Settings): Try[EasyMetadata] = {
```

\<edit afterwards> perhaps also change the name of the function that is called there accordingly.